feat: add reference for wsl profiles (#134)
canonical/landscape-documentation: docs/reference/terms/profiles.md

(reference-terms-wsl-profile)=
## WSL profile

A **WSL profile** defines what Ubuntu WSL instances should be installed on Windows instances associated with a specific access group. You can associate zero or more Windows instances with each WSL profile via tags to install Ubuntu WSL instances on those Windows instances. Each WSL profile defines a single Ubuntu WSL instance that should be present on associated Windows instances. The profile can specify which Ubuntu distribution to install, either from the Windows store or from a custom image, optionally configured with with cloud init. You can also configure the WSL profile to be **strict** which requires the Windows instance to only have Ubuntu WSL instance that are created through Landscape.

> WSL profiles are evaluated periodically for **compliance**. A Windows instance is deemed to be **compliant** if both of the following are true:

Reviewer: ```suggestion
WSL profiles are evaluated periodically for compliance. A Windows instance is considered **compliant** if the following are true:
```
- I think bolding the second is sufficient. I try to limit the bolding in general because it can get to be too much sometimes
- Simplified language
- Removed "both" - this makes it more maintainable w/o sacrificing the language anyway. (Like if another option got added later, it won't look stale if that person misses changing "both")
